mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -630,3 +633,70 @@ Feature: Loans tests
     And match postCheckInResponse.response.loan.status.name == 'Closed'
     And match postCheckInResponse.response.item.id == itemId
     And match postCheckInResponse.response.item.status.name == 'Available'
+
+  Scenario: When an overdue item is returned, an overdue fine is billed per the Overdue Fine Policy
+
+    * def extItemBarcode = 'FAT-1017IBC'
+    * def extUserBarcode = 'FAT-1017UBC'
+    * def extCheckInDate = '2021-11-17T13:30:46.000Z'
+    * def extDueDate = '2021-11-17T13:25:46'
```

Comment:
redundant

Reply:
could you tell me why they are redundant? if i won't provide itembarcode, then I can't checkout and check in

Reply:
Could you tell me where did you see any comments about itemBarcode ?

Reply:
you wrote redundant and I saw few lines and thought those vars on those lines were redundant?